### PR TITLE
DCS-1799 integration tests can stub multiple user roles

### PR DESCRIPTION
**Pull request by GurnankCheema**
need to add multiple roles as the dpslink from pre-arrival prsioner summary has a button to the DPS profile and the button should show only if user has the two required roles

### CODE REVIEW

File: integration_tests/mockApis/auth.ts [hunk 1]
```diff
@@ -5,12 +5,12 @@ import { getRequests, stubFor } from './wiremock'
 import tokenVerification from './tokenVerification'
 import Role from '../../server/authentication/role'
 
-const createToken = (role: Role) => {
+const createToken = (role: Role[]) => {
```
Comment (andrewrlee) [inline integration_tests/mockApis/auth.ts:8]: `roles` would probably be a better name

File: integration_tests/mockApis/auth.ts [hunk 2]
```diff
@@ -5,12 +5,12 @@ import { getRequests, stubFor } from './wiremock'
...
   const payload = {
     user_name: 'USER1',
     scope: ['read'],
     auth_source: 'nomis',
-    authorities: [role || ''],
+    authorities: role,
```
Comment (GurnankCheema) [inline integration_tests/mockApis/auth.ts:13]: didn't think there is there need for || ''

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: integration_tests/mockApis/auth.ts (hunk 1 above)
Comment (GurnankCheema) [reply integration_tests/mockApis/auth.ts]: pluralised variable. Left type singular

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**